release-calendar/src/server/repository-service.ts

```diff
@@ -28,6 +28,13 @@ export class RepositoryService {
     this.promotionRepository = connection.getRepository(PromotionEntity);
   }
 
+  countRelease(name: string): Promise<number> {
```

Comment:
Since `release.name` is a primary key (must be unique). This will only return a 1 or 0.